plugins/synapticsmst/synapticsmst-device.c

```diff
@@ -61,6 +76,9 @@ synapticsmst_device_kind_to_string (SynapticsMSTDeviceKind kind)
 const gchar *
 synapticsmst_device_board_id_to_string (SynapticsMSTDeviceBoardID board_id)
 {
+	#if MSTDBG
```

Comment:
It would be better to the use the built in plugin verbose functionality for this.  You can see an example of how this is done for another plugin here:
https://github.com/hughsie/fwupd/blob/master/plugins/dell/fu-plugin-dell.c#L1037 and here https://github.com/hughsie/fwupd/blob/872ec1b68f6ed644a641f18a4d99fd824c886126/plugins/wacomhid/fu-wac-common.c#L78

Reply:
Actually this stuff would be fine  to just put under `g_debug`.  That will hide it in normal case, and if daemon is launched in verbose it will show it in daemon log.